gammapy/maps/region.py

```diff
@@ -415,3 +415,29 @@ def union(self, other):
                 self._region = self.region.union(other.region)
             else:
                 self._region = other.region
+
+    def plot_region(self, ax=None, **kwargs):
+        """Plot region in the sky.
+
+        Parameters
+        ----------
+        ax : `~astropy.vizualisation.WCSAxes`
+            Axes to plot on. If no axes are given,
+            the region is shown using the minimal
+            equivalent WCS geometry.
+        **kwargs : dict
+            Keyword arguments forwarded to `~regions.PixelRegion.as_artist`
+        """
+        import matplotlib.pyplot as plt
+        from matplotlib.collections import PatchCollection
+
+        if ax is None:
+            wcs_geom = self.to_wcs_geom()
+            m = Map.from_geom(wcs_geom)
+            fig, ax, cbar = m.plot(add_cbar=False)
+        regions = compound_region_to_list(self.region)
+        artists = [region.to_pixel(wcs=ax.wcs).as_artist() for region in regions]
+
```

Comment:
Maybe set `kwargs.setdefault("fc", "None")` here, so that the region is not filled? Might be the better default, if users plot on sky images...

Reply:
I also added a default edge color because else the region was not visible when plotted alone